### PR TITLE
Cartopy res110m data

### PR DESCRIPTION
**Pull request by pmreyes2**
Downloading 110m cartopy data for features: physical cultural cultural-extra
The data weighs about 4 MB and includes among others coastline, country borders and states. This allows you to run cartopy code without the need of connectivity to internet and also when the cartopy server is down.

### CODE REVIEW

File: resen-core/Dockerfile [hunk 1]
```diff
@@ -123,3 +119,9 @@ RUN /bin/bash -c 'source /opt/conda/etc/profile.d/conda.sh && \
 RUN /bin/bash -c 'echo "alias updatedb=\"/usr/bin/updatedb --require-visibility 0 -o ~/.locate.db\"" >>  ~/.bashrc && \
     echo "alias locate=\"/usr/bin/locate -d ~/.locate.db\"" >>  ~/.bashrc && \
     /usr/bin/updatedb --require-visibility 0 -o ~/.locate.db'
+
+# Download 110m scale cartopy data
+COPY resources/feature_download.py .
+RUN /bin/bash -cl 'source activate py36 && \
```
Comment (asreimer) [inline resen-core/Dockerfile:130]: Does this download the features for both py27 and 36?

Comment (pmreyes2) [reply resen-core/Dockerfile:130]: Yes, it does and stores the data inside `~/.local/share/cartopy...`, which is where cartopy looks for pre-downloaded data for the current user regardless of the environment.

Comment (asreimer) [reply resen-core/Dockerfile:130]: Sounds good.

**Review by asreimer — APPROVED** [review]

**Comment by asreimer** [comment]
Seems to be useful from a "get up and running ASAP" perspective. We can always remove later if proves not useful.

**Pull request merged**